src/drivers/gps/gps.cpp

```diff
@@ -721,21 +714,6 @@ GPS::task_main()
 				memset(&_report_gps_pos, 0, sizeof(_report_gps_pos));
 
 				if (_mode == GPS_DRIVER_MODE_UBX) {
-					/* Publish initial report that we have access to a GPS,
```

Comment:
This needs commander inspection to make sure there are no surprises with the GPS pre-arm checks. They are disabled by default, so you would not have run into these.